inst/stan/wwinference.stan

```diff
@@ -290,9 +292,11 @@ model {
   eta_growth ~ std_normal();
   initial_growth ~ normal(initial_growth_prior_mean, initial_growth_prior_sd);
   inv_sqrt_phi_h ~ normal(inv_sqrt_phi_prior_mean, inv_sqrt_phi_prior_sd);
-  sigma_ww_site_mean ~ normal(sigma_ww_site_prior_mean_mean, sigma_ww_site_prior_mean_sd);
-  sigma_ww_site_sd ~ normal(sigma_ww_site_prior_sd_mean, sigma_ww_site_prior_sd_sd);
-  sigma_ww_site_raw ~ std_normal();
+  mode_sigma_ww_site ~ normal(mode_sigma_ww_site_prior_mode,
+                              mode_sigma_ww_site_prior_sd);
+  sd_log_sigma_ww_site ~ normal(sd_log_sigma_ww_site_prior_mode,
+                                sd_log_sigma_ww_site_prior_sd);
+  eta_log_sigma_ww_site ~ std_normal();
```

Comment:
This is a bit of a change in the actual implementation, correct?

You are now estimating the deviations on the log scale rather than natural scale?

Reply:
Yes. Doing it on the natural scale will cause bounds problems

Reply:
See my question above. Just want to make sure the priors are somewhat comparable even as we switch the form of the distribution?

Reply:
Resolved face to face that for now we'll use `sd_log_sigma_ww_site_prior_sd` ~= log(2) and reconsider default priors later